app-services/secrets/res/configuration.toml

```diff
@@ -52,17 +52,43 @@ Host = "localhost"
 Port = 27017
 Timeout = "30s"
 
+# Shared Secret Store
 [SecretStore]
-  Host = 'localhost'
+  # Host = 'localhost'
+  Host = 'edgex-vault'
   Port = 8200
-  Path = '/v1/secret/edgex/appservice'
-  Protocol = 'http'
-  Tokenfile = 'res/token.json'
+  Path = '/v1/secret/edgex/edgex-appservice/'
+  Protocol = 'https'
+  RootCaCertPath = '/tmp/edgex/secrets/ca/ca.pem'
+  ServerName = 'edgex-vault'
+  TokenFile = '/tmp/edgex/secrets/edgex-appservice/secrets-token.json'
+  # TokenFile = './res/token.json'
+  # Number of attempts to retry retrieving secrets before failing to start the service.
+  AdditionalRetryAttempts = 10
+  # Amount of time to wait before attempting another retry
+  RetryWaitPeriod = "1s"
 
   [SecretStore.Authentication]
-  AuthType = 'X-Vault-Token'
-  # Vault root token that's used for authentication
-  AuthToken = 's.imTIVMl1OWhlX3zMAhSyaFlO'
+    AuthType = 'X-Vault-Token'
+
+# Exclusive Secret Store
+[SecretStoreExclusive]
+  # Host = 'localhost'
+  Host = 'edgex-vault'
+  Port = 8200
+  Path = '/v1/secret/edgex/secrets-example/'
+  Protocol = 'https'
+  RootCaCertPath = '/tmp/edgex/secrets/ca/ca.pem'
+  ServerName = 'edgex-vault'
```

Comment:
Change to `localhost` and use override in compose file specific to the app service.